README.md

```diff
@@ -0,0 +1,5 @@
+
+# oCam Camera driver
+Did some preprocessing using Histogram equalization and Undistortion
+
+Relevant link: https://github.com/withrobot/oCam/tree/master/Software/oCam_ROS_Package/ocam
```

Comment:
Mention that this is a fork of the main driver, but we have only lifted the ROS interface

Reply:
Done